### PR TITLE
feat: [CCM-14434] API support for CRUD of fixed schedule of AutoStopping

### PR DESCRIPTION
**Pull request by shalinlk**
Ticket : [CCM-14434](https://harness.atlassian.net/browse/CCM-14434)

## Change
* API changes for CRUD of fixed schedule feature of AutoStopping

## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`

[CCM-14434]: https://harness.atlassian.net/browse/CCM-14434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sujeesh-harness — APPROVED** [review]

File: harness/nextgen/api_cloud_cost_auto_stopping_fixed_schedules.go [hunk 1]
```diff
@@ -208,18 +193,17 @@ func (a *CloudCostAutoStoppingFixedSchedulesApiService) DeleteAutoStoppingSchedu
...
 
 	if localVarHttpResponse.StatusCode >= 300 {
 		newErr := GenericSwaggerError{
 			body:  localVarBody,
 			error: localVarHttpResponse.Status,
 		}
+
 		if localVarHttpResponse.StatusCode == 200 {
-			var v InlineResponse2001
+			var v FixedSchedulesGetResponse
```
Comment (sandyydk) [inline harness/nextgen/api_cloud_cost_auto_stopping_fixed_schedules.go:206]: Does this replace generic response with a specific model? Am I correct in reading it that way? @shalinlk

**Review by sandyydk — APPROVED** [review]
Looks good.

File: harness/nextgen/api_cloud_cost_auto_stopping_fixed_schedules.go (hunk 1 above)
Comment (shalinlk) [reply harness/nextgen/api_cloud_cost_auto_stopping_fixed_schedules.go:206]: It is. It was an inline response which I made into a proper response structure in swagger

File: harness/nextgen/api_cloud_cost_auto_stopping_fixed_schedules.go [hunk 2]
```diff
@@ -73,19 +72,6 @@ func (a *CloudCostAutoStoppingFixedSchedulesApiService) CreateAutoStoppingSchedu
 	}
 	// body params
 	localVarPostBody = &body
-	if ctx != nil {
-		// API Key Authentication
-		if auth, ok := ctx.Value(ContextAPIKey).(APIKey); ok {
-			var key string
-			if auth.Prefix != "" {
```
[the remark this answers is not in the document]
Comment (shalinlk) [reply harness/nextgen/api_cloud_cost_auto_stopping_fixed_schedules.go:80]: This is auto-generated @sandyydk . Thought of not touching it

**Review by riyasyash — COMMENTED** [review]
Looks good

**Pull request merged**